modules/tinymce/src/core/test/ts/browser/selection/SelectionBookmarkInlineEditorTest.ts

```diff
@@ -60,6 +60,19 @@ describe('browser.tinymce.core.selection.SelectionBookmarkInlineEditorTest', ()
   before(() => addTestDiv());
   after(() => removeTestDiv());
 
+  it('assert bookmark is updated in response to `setRng`', () => {
+    const editor = hook.editor();
+    editor.setContent('<p>a</p><p>b</p>');
+    // In FireFox blurring the editor adds an undo level that triggers a nodechange that creates a bookmark,
+    // so by adding an undo level first we keep it from adding a bookmark because the undo manager
+    // does not add a new undolevel if it is the same as the previous level.
+    editor.undoManager.add();
```

Comment:
```suggestion
    editor.resetContent('<p>a</p><p>b</p>');
```
This can just be replaced with this I believe instead, as the issue IIUC is that `setContent` doesn't create an undo level, so when it blurs it detects the editor content has changed and creates one.

Reply:
this was just copied from the other test, I'll change both

Reply:
Yeah I did notice that and thanks! Fwiw that test was created before the `resetContent` API was added, so that's likely why it did that back then.